src/line_shapes/get_line_shapes.jl

```diff
@@ -0,0 +1,141 @@
+
+"""
+Code to measure line bisectors and widths as a function of depth.
+Author: Alex Wise
+Created: September 2020
+"""
+
+""" get_line_shapes(line_list, clt, depths, v_width)
+For each spectrum in clt (chunklist timerseries), returns data frames for
+line bisector and width as a function of fractional line depth given in frac_depths
+note: frac_depths must be sorted from largest to smallest
+"""
+function get_line_shapes(line_list::A1, clt::AbstractChunkListTimeseries;
+                    frac_depths::A2 = 0.99:-0.01:0.05, v_width::Real=15e3,
+                    RV_estimate::Real=-5000.0) where { T1<:Real, A1<:AbstractArray{T1,1},
+                    T2<:Real, A2<:AbstractArray{T2,1}}
+    ll = copy(line_list)
+    n_lines = length(ll)
+    n_spectra = length(clt.chunk_list)
+    n_chunks = length(clt.chunk_list[1].data) # this assumes the number of chunks in every spectrum is the same
+    n_depths = length(frac_depths)
+    #allocate memory for line shapes
+    bisectors = zeros(n_lines,n_depths,n_spectra)
```

Comment:
Is there a reason for this order?  I'd guess that (n_depths,n_lines,n_spectra) or (n_depths,n_spectra,n_lines) is more likely to be contiguous memory accesses.